src/lxc/api_extensions.h

```diff
@@ -43,6 +43,7 @@ static char *api_extensions[] = {
 	"cgroup_relative",
 	"mount_injection_file",
 	"seccomp_allow_nesting",
+	"seccomp_notify",
```

Comment:
@brauner is it worth adding an entry to https://github.com/lxc/lxc/blob/master/doc/api-extensions.md for this feature?

Reply:
yeah